assets/dev/scss/global/_flatpicker.scss

```diff
@@ -1,26 +0,0 @@
-.flatpickr-calendar {
-	width: 280px;
-
-	.flatpickr-current-month {
-
-		span.cur-month {
-			font-weight: 300;
-		}
-	}
-
-	.dayContainer {
-		width: 280px;
-		min-width: 280px;
-		max-width: 280px;
-	}
-
-	.flatpickr-days {
-		width: 280px;
-	}
-
-	.flatpickr-day {
-		max-width: 37px;
-		height: 37px;
-		line-height: 37px;
-	}
-}
```

Comment:
@arielk Why did you deleted this style?

Reply:
@ramiy Is no longer relevant, I replaced it with few customization in the original CSS file see here:
https://github.com/elementor/elementor/pull/28007/files#diff-2fb1861c7b79dc80134e146fb49de34ef28aa8ac58c125e652e1aab3495a8b15R295